index.js

```diff
@@ -265,19 +265,25 @@ function log (message, site) {
  */
 
 function callSiteLocation (callSite) {
-  var file = callSite.getFileName() || '<anonymous>'
-  var line = callSite.getLineNumber()
-  var colm = callSite.getColumnNumber()
-
-  if (callSite.isEval()) {
-    file = callSite.getEvalOrigin() + ', ' + file
+  var functionName, site
+  if (callSiteLocation) {
```

Comment:
Isn't this always `true`? `callSiteLocation` is a `function` that is... always defined?

Reply:
I have juste spotted this in my tests and fixed it!

Reply:
My test was basically running https://github.com/voyages-sncf-technologies/nodejs-depd-bug-repro.git

The use of `nvm` and 2 different NodeJS versions makes it non-trivial to integrate in `depd` current test suite.
That's why I suggested to simply add a unit test that applies the same mock as `core-js`